libs/blockchain/migrations/6_proxy_clone_factory.js

```diff
@@ -0,0 +1,11 @@
+const ProxyCloneFactory = artifacts.require("ProxyCloneFactory");
+const ProxyV2 = artifacts.require("ProxyV2");
+
+module.exports = async function (deployer, network, addresses) {
+  deployer.deploy(ProxyCloneFactory);
```

Comment:
This contract is ownable, who will wind up being the owner? Should we throw ownership away?

Reply:
I'm using the same mnemonic that was used to run migrations in `celo-monorepo` but we should indeed make an explicit decision about ownership here.